README.md

```diff
@@ -38,7 +38,7 @@ To enable support for older browsers that don’t support CSS variables natively
 To apply a specific theme on top of the library components, the OneUI utility can be used to parse the provided theme file:
 
 ```javascript
-import OneUI from '@textkernel/oneui';
+import { OneUI } from '@textkernel/oneui';
```

Comment:
I wonder if this should not be an exception to this rule. If we change this export it would be a breaking change, and also it would need an other name if we want to be named export.

Reply:
I'm inclined to agree with that